lib/gstreamer/msdk/encoder.py

```diff
@@ -63,12 +63,13 @@ def maxrate(self):
   ladepth = property(lambda s: s.ifprop("ladepth", " rc-lookahead={ladepth}"))
   tilecols = property(lambda s: s.ifprop("tilecols", " num-tile-cols={tilecols}"))
   tilerows = property(lambda s: s.ifprop("tilerows", " num-tile-rows={tilerows}"))
+  maxFrameSize = property(lambda s: s.ifprop("maxFrameSize", " max-frame-size={maxFrameSize}"))
```

Comment:
Camel case name doesn't match general style.  How about `max_frame_size` or `maxfrsize` or `maxframesize`?

Reply:
> Camel case name doesn't match general style. How about `max_frame_size` or `maxfrsize` or `maxframesize`?

@uartie thanks for your advice. I will modify later.